resource-aggregate/pb/commands.proto

```diff
@@ -385,3 +385,96 @@ message ConfirmResourceCreateResponse {
+//     │ │                   │                                      │ │ ──────────────────────────────────────────────────>│ │  
+//     │ │                   │                                      └┬┘                            │                       │ │  
+//     │ │                   │                                      │ DeviceMetadataUpdatePending  │                       │ │  
+//     │ │                   │                                      │  ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─>│                       │ │  
+//     │ │                   │                                      │                              │                       │ │  
+//     │ │                  ┌┴┐                     DeviceMetadataUpdatePending                    │                       │ │  
+//     │ │                  │ │ <─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─                       │ │  
+//     │ │                  │ │                                     │                              │                       │ │  
+//     │ │                  │ │ ConfirmDeviceMetadataUpdateRequest  ┌┴┐                            │                       │ │  
+//     │ │                  │ │ ───────────────────────────────────>│ │                            │                       │ │  
+//     │ │                  └┬┘                                     │ │                            │                       │ │  
+//     │ │                   │ConfirmDeviceMetadataUpdateResponse   │ │                            │                       │ │  
+//     │ │                   │<──────────────────────────────────── │ │                            │                       │ │  
+//     │ │                   │                                      └┬┘                            │                       │ │  
+//     │ │                   │                                      │    DeviceMetadataUpdated     │                       │ │  
+//     │ │                   │                                      │  ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─>│                       │ │  
+//     └┬┘                   │                                      │                              │                       └┬┘  
+//      │                    │                                      │                              │ DeviceMetadataUpdated │    
+//      │                    │                                      │                              │  ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─>    
+// ┌──────────┐          ┌───┴───┐                         ┌────────┴─────────┐               ┌─────────┐               ┌──────┐
+// │OCF Server│          │Gateway│                         │Resource Aggregate│               │Event Bus│               │Client│
+// └──────────┘          └───────┘                         └──────────────────┘               └─────────┘               └──────┘
+message ShadowSynchronizationStatus {
+    bool disabled = 1;
```

Comment:
I am for isEnabled :)

Reply:
it about default value at proto it is `false` so if i changed it then all  must set it to true ... eg coap-gw but we don't want to do it ...